README.md

```diff
@@ -57,28 +59,106 @@ systemctl restart jans-auth
 
 ### Deployment
 
-Download the
-latest [agama-passkey.gama](https://github.com/GluuFederation/agama-passkey/releases/latest/download/agama-passkey.gama)
-file and deploy it in Auth Sever.
+Run these instructions on the server where you have `Janssen` or `Gluu` installed:
 
-Follow the steps below:
-
-- Copy (SCP/SFTP) the gama file of this project to a location in your `Jans Server`
-- Connect (SSH) to your `Jans Server` and open TUI: `python3 /opt/jans/jans-cli/jans_cli_tui.py`
-- Navigate to the `Agama` tab and then select `"Upload project"`. Choose the gama file
+- Download the
+  latest [agama-passkey.gama](https://github.com/GluuFederation/agama-passkey/releases/latest/download/agama-passkey.gama), you can use `wget`
+```
+wget https://github.com/GluuFederation/agama-passkey/releases/latest/download/agama-passkey.gama
+```
+- Open TUI: `python3 /opt/jans/jans-cli/jans_cli_tui.py`
+- Navigate to the `Agama` tab and then select `"Upload project"`. Choose the `.gama` file
 - Wait for about one minute and then select the row in the table corresponding to this project
-- Press `d` and ensure there were not deployment errors
+- Press `v` and ensure there were not deployment errors
 - Pres `ESC` to close the dialog
 
 ![TUI_AGAMA_DEPLOY](https://github.com/GluuFederation/agama-passkey/assets/86965029/1d6b8cab-ddad-451c-b620-d19be1b7f9e3)
 
 ### Configure Jans Scim
 
-- Once we have deployed the `agama project`, we need to configure the `jans scim` parameters, then we proceed to create a new user `jans scim` with scope `https://jans.io/scim/fido2.read` and `https://jans.io/scim/fido2.write`.
+- Once we have deployed the `agama project`, we need to configure the `jans scim` parameters, then we proceed to create
+  a new client `jans scim` with scope `https://jans.io/scim/fido2.read` and `https://jans.io/scim/fido2.write`.
+
+You can create the client using the registration web service.
+
+**Request**
+
+```
+curl --location 'https://YOUR_DOMAIN/jans-auth/restv1/register' \
+--header 'Content-Type: application/json' \
+--data '{
+  "client_name": "Scim Agama Client",
+  "scope": [
+    "https://jans.io/scim/fido2.read",
+    "https://jans.io/scim/fido2.write"
+  ],
+  "grant_types": [
+    "client_credentials"
+  ],
+  "token_endpoint_auth_method": "client_secret_basic",
+  "response_types": [
+    "code"
```

Comment:
why does a backchannel client need response_type=code? It doesn't... remove response_type or send nothing.